scripts/superdesk-monitoring/monitoring.js

```diff
@@ -535,7 +542,12 @@
                             if (scope.total !== items._meta.total) {
                                 scope.total = items._meta.total;
                             }
-
+                            // For highlight page return only highlight items, i.e, include only last version if item type is published
+                            if (scope.group.type === 'highlights') {
```

Comment:
This section looks identical to one on line:529 so it'd be better to factor both to its own function.